### ISSUE TITLE
Cors

### ISSUE THREAD

**Issue by josjojo** [hedwiggggg/ugrip#5]
When I press 'Load song' nothing happens. In the developer tools console of Firefox I see the error:
> Cross-Origin-request blocked
With the url:
> http://0.0.0.0:5001/https://tabs.ultimate-guitar.com/tab/hillsong-united/heart-of-worship-chords-1012850

I have tried to install a CORS enabler add-on, but it did not solve the problem.

**Comment by NRSC** [comment]
Pleb JS user also having the same troubles.

**Comment by hedwiggggg** [comment]
Oh, hi, sorry I didn't saw that someone created an issue. 
Nice github, thanks for notifying.

So the main issue is, that you cannot directly access ultimate-guitar, because their server response doesn't include any relevant 'Access-Control-Allow-Origin' headers. So the browser will block the request.

**Comment by hedwiggggg** [comment]
> When I press 'Load song' nothing happens. In the developer tools console of Firefox I see the error:
> 
> > Cross-Origin-request blocked
> > With the url:
> > http://0.0.0.0:5001/https://tabs.ultimate-guitar.com/tab/hillsong-united/heart-of-worship-chords-1012850
> 
> I have tried to install a CORS enabler add-on, but it did not solve the problem.

So whats basically happening here, the UI tries to access the cors-anywhere server located @ http://localhost:5001 which is not existing.

**Comment by hedwiggggg** [comment]
Okay, I've pusehd a new version, that should be simpler to use. 
I added a default .env with `https://api.codetabs.com/v1/proxy/?quest=` as the cors proxy.

As long as the don't shut down there service it should work out of the box.

I also changed the

```
REACT_APP_CORS_HOST
REACT_APP_CORS_PORT
REACT_APP_CORS_PROTOCOL
```

structure to only using `REACT_APP_CORS_SERVER`

So if they ever shut down their service, just replace the `REACT_APP_CORS_SERVER` url with a new proxy :)
